matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/message/StickerJsonMessage.java

```diff
@@ -0,0 +1,25 @@
+/* 
+ * Copyright 2014 OpenMarket Ltd
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.matrix.androidsdk.rest.model.message;
+
+public class StickerJsonMessage {
```

Comment:
Could we have a comment for this class?
Explaining that it is just here as an intermediate object to create a StickerMessage from a m.sticker event type.